src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -4242,6 +4242,18 @@ public final CommandObject<Object> tFunctionCallAsync(String library, String fun
   }
   // RedisGears commands
 
+  public final CommandObject<Object> latencyHistory(Protocol.EventKeyword keyword) {
+    return new CommandObject<>(commandArguments(LATENCY).add(HISTORY).add(keyword), BuilderFactory.RAW_OBJECT);
+  }
+
+  public final CommandObject<Long> latencyReset(Protocol.EventKeyword ... events) {
+    CommandArguments arguments = commandArguments(LATENCY).add(Keyword.RESET);
+    for (Protocol.EventKeyword event: events) {
+      arguments.add(event);
+    }
+    return new CommandObject<>(arguments, BuilderFactory.LONG);
+  }
+
```

Comment:
Remove these from here. Do the works in Jedis class.

Reply:
remove code from CommandObjects, now all works in Jedis class